### ISSUE TITLE
 OsTicket and nginx not work properly

### ISSUE THREAD

**Issue by LinuxTik** [osTicket/osTicket#3402]
I can not see the online help or attach files on tickets.

![error_ost1](https://cloud.githubusercontent.com/assets/22861600/19413806/207e7870-9339-11e6-874c-02b22c43aaae.png)

![error_ost](https://cloud.githubusercontent.com/assets/22861600/19413808/3bab1cac-9339-11e6-9379-5d11b05eb935.png)

Log's show the nginx server:

`2016/10/16 00:28:38 [error] 17688#0: *1075 open() "/var/www/osticket/upload/ajax.php/config/client" failed (20: Not a directory), client: x.x.x.x, server: x.x.x.x, request: "GET /ajax.php/config/client HTTP/1.1", host: "www.domain.com", referrer: "https://www.domain.com/open.php"`

`2016/10/16 00:28:43 [error] 17688#0: *1075 open() "/var/www/osticket/upload/ajax.php/form/upload/attach" failed (20: Not a directory), client: X.X:X:X, server: www.domain.com, request: "POST /ajax.php/form/upload/attach HTTP/1.1", host: "www.domain.com", referrer: "https://www.domain.com/open.php"``

I followed the directions tomashastings but still the error in version v1.9.14:

https://github.com/osTicket/osTicket-1.7/issues/538

Any help?, solution?

Thanks very much.

**Comment by ntozier** [comment]
nginx is not a support web server at this time.  
Please see the numerous threads here that discuss nginx (by searching for nginx)
Here are some that might assist you:
https://github.com/osTicket/osTicket/issues/2797
https://github.com/osTicket/osTicket-1.7/issues/538
https://github.com/osTicket/osTicket/issues?q=is%3Aissue+is%3Aopen+nginx+label%3Anginx
